front/php/templates/footer.php

```
@@ -43,6 +43,9 @@
 </div>
 <!-- ./wrapper -->
 
+<!-- Modals — placed outside .wrapper so position:fixed modals are in the root stacking context -->
+<?php require 'php/templates/modals.php'; ?>
```

PHPStan (2.2.2)

[error] Path in require() "php/templates/modals.php" is not a file or it does not exist.


(require.fileNotFound)